index.bs

```diff
@@ -847,16 +847,12 @@ these steps.
        1. [=Cleanup=] |transport| with |error|, |error| and true.
        1. Abort these steps.
      1. Let |session| be |transport|'s [=[[Session]]=].
+     1. Let |code| be |closeInfo|.{{WebTransportCloseInfo/closeCode}}.
      1. Let |reason| be an empty [=byte sequence=].
-     1. If |closeInfo|.{{WebTransportCloseInfo/reason}} exists, set |reason| be
-        |closeInfo|.{{WebTransportCloseInfo/reason}}, [=UTF-8 encoded=].
+     1. Set |reason| to |closeInfo|.{{WebTransportCloseInfo/reason}}, [=UTF-8 encoded=].
      1. If |reason|'s [=byte sequence/length=] exceeds 1024, then set |reason| to a
         [=byte sequence/prefix=] of |reason| whose length is 1024.
```

Comment:
(Side-observation from looking at the wpt test) Doesn't this risk clipping a multi-byte UTF8 sequence?

Reply:
Issue?

Reply:
Filed as https://github.com/w3c/webtransport/issues/382